src/android/java/org/alias/wallet/BootstrapService.java

```diff
@@ -7,33 +7,41 @@
 import android.content.Intent;
 import android.graphics.drawable.Icon;
 import android.os.AsyncTask;
-import android.os.Bundle;
 import android.os.IBinder;
 import android.util.Log;
 
 import com.google.android.gms.net.CronetProviderInstaller;
 import com.google.android.gms.tasks.Task;
 
+import net.lingala.zip4j.ZipFile;
+import net.lingala.zip4j.exception.ZipException;
+
 import org.chromium.net.CronetEngine;
 import org.chromium.net.impl.JavaCronetProvider;
 
 import java.io.BufferedInputStream;
+import java.io.BufferedReader;
 import java.io.Closeable;
 import java.io.File;
 import java.io.FileOutputStream;
+import java.io.FileReader;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.HttpURLConnection;
-import java.net.MalformedURLException;
 import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.security.DigestInputStream;
+import java.security.MessageDigest;
+import java.time.Clock;
+import java.time.Duration;
+import java.util.ArrayList;
 import java.util.Comparator;
-import java.util.Enumeration;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipFile;
+import java.util.List;
+
+import static org.alias.wallet.BootstrapService.BootstrapError.*;
```

Comment:
Why this import? The enum is defined inside of the class!?

Reply:
Without the static import, you have to prefix the enum constants with the enum type when using them

Reply:
True, ok.